.eslintrc.json

```diff
@@ -11,6 +11,7 @@
         "it": true
       },
       "rules": {
+        "no-undef": "off",
```

Comment:
Why turn this off?

Reply:
Oop, that's me being lazy, I'll set up globals for Mocha.